nilearn/decoding/space_net.py

```diff
@@ -614,11 +614,31 @@ class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
 
     Attributes
     ----------
-    `alpha_` : float
-         Best alpha found by cross-validation.
+    
+    `all_coef_` : ndarray, shape (n_l1_ratios, n_folds, n_features)
+        Coefficients for all folds and features.
+    
+    `alpha_grids_` : ndarray, shape (n_folds, n_alphas)
+        Alpha values tried for choosing the best ones (saved in best_model_params_)
+
+    `best_model_params_` : ndarray, shape (n_folds, n_parameter)
+        Best model parameters (alpha, l1_ratio) saved for the different 
+        cross-validation folds.
+
+    `classes_` : array of labels (n_classes_)
+        Labels of the classes
+
+    `n_classes_` : int
+        Number of classes
 
     `coef_` : ndarray, shape (n_classes-1, n_features)
         Coefficient of the features in the decision function.
+    
+    `coef_img_` : nifti image
+        On mask model coefficients
```

Comment:
I think it's a typo. It should indeed be something like: "On mask model
coefficients"

On Thu, Dec 3, 2015 at 12:03 PM, Danilo Bzdok <notifications@github.com>
wrote:

> In nilearn/decoding/space_net.py
> <https://github.com/nilearn/nilearn/pull/881#discussion_r46538268>:
>
> >
> >      `coef_` : ndarray, shape (n_classes-1, n_features)
> >          Coefficient of the features in the decision function.
> > +
> > +    `coef_img_` : nifti image
> > +        On mask model coefficients
>
> What does "On mask model coefficients" mean, it may not be clear to
> everybody. Perhaps instead: Masked model coefficients?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nilearn/nilearn/pull/881/files#r46538268>.
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob